docker-build/build_docker.sh

```diff
@@ -92,20 +92,18 @@ function packaging_fate() {
 
 function packaging_fateflow() {
     echo "[INFO] package fateflow start"
-    #pull_fateflow
+    pull_fateflow
     cp -r fateflow ${package_dir}/
     echo "[INFO] package fateflow done"
 }
 
 packaging_fateboard() {
     echo "[INFO] package fateboard start"
-    #pull_fateboard
+    pull_fateboard
     cd ./fateboard
     fateboard_version=$(grep -E -m 1 -o "<version>(.*)</version>" ./pom.xml | tr -d '[\\-a-z<>//]' | awk -F "version" '{print $2}')
     echo "[INFO] fateboard version "${fateboard_version}
-    
-    docker run --rm -u $(id -u):$(id -g) -v ${source_dir}/fateboard/resources-front-end:/data/projects/fate/fateboard/resources-front-end node:16 /bin/bash -c "cd /data/projects/fate/fateboard/resources-front-end && npm config set registry https://registry.npm.taobao.org && npm install && npm rebuild node-sass && npm run build"
-    
```

Comment:
Why remove this?

Reply:
`npm run build` is included in the `mvn clean package -DskipTests` step. So there is no need here.

https://github.com/FederatedAI/FATE-Board/blob/c199bc311315d472a289fcb81adf2328e8233ef0/pom.xml#L245-L289